TemporaryOysters/bangazon-cli
# Me fixtures

##Pull Requests



##Descriptions:

* Please describe what issue this PR solves
Database is able to be created and loaded with fixtures
#8 



There are a couple of places where code needs to to be commented in/out after tables are created. Please see steps to test.







##Steps to Test:


Fetch the latest branches
  1. Checkout `me-fixtures`
  2. From terminal run `python main.py`, which will create the initial tables
  3. Verify this using DB Browser -From within DB Browser open `bangazon.db` file located in bangazon-cli folder and verify tables were created
  4. Once verified, open `main.py`, ---> comment out line 7, ---> comment in line 9
  5. From terminal run `python main.py`, which will populate the tables
  6. Verify by repeating step 3
  7. Pour yourself a cold one. You've just created a table with fixtures.

##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?
No 3rd party libraries necessary

* Any command line utilities to run?
See above
